OpenRA.Platforms.Default/FrameBuffer.cs

```diff
@@ -109,6 +113,25 @@ public void Unbind()
 			OpenGL.CheckGLError();
 		}
 
+		public void EnableScissor(Rectangle rect)
+		{
+			VerifyThreadAffinity();
+
+			OpenGL.glScissor(rect.X, rect.Y, Math.Max(rect.Width, 0), Math.Max(rect.Height, 0));
+			OpenGL.CheckGLError();
```

Comment:
Do we need to use `CheckGLError` after #16984?

Reply:
We do, yeah - not all platforms support the debug message handler.